.github/workflows/main.yml

```diff
@@ -6,13 +6,11 @@ name: CI
 on:
   # Triggers the workflow on push or pull request events but only for the master branch
   push:
-    paths-ignore:
-      - '.circleci'
+    branches:
+      - '*'
```

Comment:
This conflicts with #5751 and will bring back test runs on push & PR for `renovate`

Reply:
This was never intended for merging: the branch didn't want to build, I just went into trial-and-error, but GitHub Actions turned out to be down.